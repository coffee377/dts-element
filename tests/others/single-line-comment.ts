import * as dts from '../index';

it('should return correctly with single-line-text', () => {
  expect(dts.emit(
    dts.create_single_line_comment({
      text: 'test',
    }),
  )).toMatchSnapshot();
});

it('should return correctly with single-line-text, leading_space', () => {
  expect(dts.emit(
    dts.create_single_line_comment({
      text: 'test',
      leading_space: true,
    }),
  )).toMatchSnapshot();
});

it('should return correctly with multi-line-text', () => {
  expect(dts.emit(
    dts.create_single_line_comment({
      text: 'test 1\ntest 2',
    }),
  )).toMatchSnapshot();
});

it('should return correctly with multi-line-text, leading_space', () => {
  expect(dts.emit(
    dts.create_single_line_comment({
      text: 'test 1\ntest 2',
      leading_space: true,
    }),
  )).toMatchSnapshot();
});
