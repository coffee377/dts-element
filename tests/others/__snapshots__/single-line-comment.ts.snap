// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return correctly with multi-line-text 1`] = `
"//test 1
//test 2"
`;

exports[`should return correctly with multi-line-text, leading_space 1`] = `
"// test 1
// test 2"
`;

exports[`should return correctly with single-line-text 1`] = `"//test"`;

exports[`should return correctly with single-line-text, leading_space 1`] = `"// test"`;
